packages/devtools_app/lib/src/screens/memory/memory_screen.dart

```diff
@@ -57,7 +58,7 @@ class MemoryBodyState extends State<MemoryBody>
         ProvidedControllerMixin<MemoryController, MemoryBody> {
   MemoryController get memoryController => controller;
 
-  late ChartControllers _chartControllers;
+  late MemoryChartPaneController _chartControllers;
```

Comment:
rename the variable to be singular instead of plural now that the class name has the singular form